packages/terra-table/src/TableSectionHeader.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import TableUtils from './TableUtils';
+import styles from './TableSectionHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Number of columns the section header must span
+   */
+  colSpan: PropTypes.number.isRequired,
+  /**
+   * Whether or not the section is collapsed.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Whether or not the section can be collapsed.
+   */
+  isCollapsible: PropTypes.bool,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
Same question as above, should `1` be an option for a11y composition?

Reply:
Updated.